readers-frontend/src/components/utils/articles.ts

```diff
@@ -3,55 +3,68 @@ import prisma from "../../prisma";
-  include: { articlesDrafts: {
-    include: { users: true },
-  } },
+  include: {
+    articlesDrafts: {
+      include: { users: true },
+    },
+  },
 });
-export type articlesWithArticleDrafts = Prisma.articlesGetPayload<typeof articlesWithArticleDrafts>
+export type articlesWithArticleDrafts = Prisma.articlesGetPayload<
+  typeof articlesWithArticleDrafts
+>;
 
 /**
  * Retrieves an article with its latest draft and the user who wrote it.
  * @param {number[]} articleIds The article IDs to retrieve.
  * @returns {Promise<articlesWithArticleDrafts[]>} Promise object represents the articles.
  */
-export const getArticles = async (articleIds?: number[]): Promise<articlesWithArticleDrafts[]> => {
+export const getArticles = async (
+  articleIds?: number[]
+): Promise<articlesWithArticleDrafts[]> => {
   if (articleIds == undefined) {
-    return []
+    return [];
   }
 
   return await prisma.articles.findMany({
     where: {
       articles_id: {
-        in: articleIds
+        in: articleIds,
       },
       articles_showInLists: true,
     },
     include: {
-      articlesDrafts: { // Get the latest draft for every featured article
+      articlesDrafts: {
+        // Get the latest draft for every featured article
         orderBy: {
           articlesDrafts_timestamp: "desc",
         },
         take: 1,
-        include: { // Get the user who wrote the article
+        include: {
+          // Get the user who wrote the article
           users: true,
-        }
+        },
       },
     },
   });
-}
+};
 
 /**
  * Retrieves an article image.
  * @param {articlesWithArticleDrafts} article The article.
  * @param {"tiny" | "small" | "medium" | "large" | false} [size] File size.
  * @returns {Promise<string>} Promise object represents the article image url.
+ * TODO: Update the image for articles with articles_displayImages as false. We're currently using the archive one.
  */
-export const getArticleImage = async (article: articlesWithArticleDrafts, size: "tiny" | "small" | "medium" | "large" | false = "large"): Promise<string> => {
-    if (!article.articles_displayImages) {
-      return import.meta.env.fileStoreUrl + "/nouseSiteAssets/imageArchive-comp.jpg";
-    } else if (Number(article.articles_thumbnail)) {
-      return await s3URL(Number(article.articles_thumbnail), size);
-    }
-    else
-      return import.meta.env.ARCHIVEFILESTOREURL + article.articles_thumbnail;
-}
+export const getArticleImage = async (
+  article: articlesWithArticleDrafts,
+  size: "tiny" | "small" | "medium" | "large" | false = "large"
+): Promise<string> => {
+  if (!article.articles_displayImages) {
+    return (
+      import.meta.env.fileStoreUrl + "/nouseSiteAssets/imageArchive-comp.jpg"
+    );
+  } else if (Number(article.articles_thumbnail)) {
+    return await s3URL(Number(article.articles_thumbnail), size);
+  } else
+    return import.meta.env.ARCHIVEFILESTOREURL + article.articles_thumbnail;
```

Comment:
The env variable is lowercase

Reply:
ahh I see I thought you meant the env in the tests. This was there from before this ticket